scripts/ci/changelog/templates/template.md.tera

```diff
@@ -19,17 +19,16 @@ This release contains the changes from `{{ env.REF1 }}` to `{{ env.REF2 }}`.
 {# We check for host function first because no matter what the priority is, #}
 {# we will force it to HIGH if at least one host function was detected. #}
 
-{% include "host_functions.md.tera" -%}
-{% include "global_priority.md.tera" -%}
-{% include "compiler.md.tera" -%}
-{% include "migrations-db.md.tera" -%}
-
 {% if env.RELEASE_TYPE and env.RELEASE_TYPE == "client" %}
-<!-- skipping runtime data for RELEASE_TYPE = {{ env.RELEASE_TYPE }} -->
+    {% include "host_functions.md.tera" -%}
+    {% include "global_priority.md.tera" -%}
+    {% include "compiler.md.tera" -%}
+    {% include "migrations-db.md.tera" -%}
+    <!-- skipping runtime data for RELEASE_TYPE = {{ env.RELEASE_TYPE }} -->
 {% else %}
-{% include "migrations-runtime.md.tera" -%}
-
-{% include "runtimes.md.tera" -%}
+    {% include "compiler.md.tera" -%}
```

Comment:
```suggestion
```
Given that runtimes are built inside docker the compiler version here is irrelevant so we should not emit it.